backend/worker/common_tlds.dict

```diff
@@ -1,369 +1,99 @@
+ac.uk
```

Comment:
Why did we change this? Is the new list better?

Reply:
This list only includes the tlds that whois supports